tests/Controller/FeedControllerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use App\Entity\Post;
+use App\Entity\Dog;
+use App\Service\DogCRUDService;
+use App\Service\PostCRUDService;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class FeedControllerTest extends WebTestCase
+{
+    public function testPostCanBeCreated(): void
+    {
+        self::bootKernel();
+
+        $dogRepo = static::getContainer()->get(DogCRUDService::class);
+        $dog = new Dog();
+        $dog->setUserName('admin');
+        $dog->setPassword('password123');
+        $dogRepo->saveDog($dog);
+
+        
+
+        $post = new Post();
+
+
+        $post->setDogId($dog->getId());
+        $post->setPostText("Woof Woof");
+        $post->setCreatedAt(new \DateTimeImmutable());
+
+        $postRepo->savePost($post);
+        echo($dog);
+        // $this->assertTrue($newDog->getUserIdentifier() === 'test123');
+        // $client->followRedirects(true);
+    }
```

Comment:
Loving the test. is there not a way to do account assumption in Symfony? That would be much more realistic in respect to what you're actually testing.

Reply:
I'd also think about the name. Does the name actually give a fair description as to what your test is checking?

Reply:
have a read of this https://symfony.com/doc/current/testing.html#logging-in-users-authentication